ngx-fudis/projects/ngx-fudis/src/lib/components/form/date/datepicker/datepicker.component.html

```diff
@@ -32,13 +32,13 @@
       class="fudis-datepicker__toggle"
       [id]="id + '-calendar-icon-toggle'"
       [for]="picker"
-      [disabled]="control.disabled"
+      [disabled]="control.disabled || disabled"
```

Comment:
Is this necessary, as FormControl is binded to the input element, so this probably should be removed?